index.js

```diff
@@ -153,29 +155,21 @@ slackEvents.on('message', event => {
         break;
       }
 
-      actions.saveLocation(event);
-      if (incidentHandlerVerification(event.text) === 'Elsewhere') {
-        sc.chat.postMessage(event.channel, '', getLocationHandlerMessage, err => {
-          if (err) {
-            logError(err);
-          }
-        });
-      } else {
-        actions.tempIncidents[userId].incidentHandler = incidentHandlerVerification(event.text);
-        actions.tempIncidents[userId].step += 1;
-        sc.chat.postMessage(event.channel, '', categoryMessage, err => {
-          if (err) {
-            logError(err);
-          }
-        });
-      }
-      break;
-    case 3:
+      actions.saveTypedLocation(event);
+
+      sc.chat.postMessage(event.channel, '', getLocationHandlerMessage, err => {
+        if (err) {
+          logError(err);
+        }
+      });
       break;
     case 4:
-      console.log('Logic flaw??'); // eslint-disable-line no-console
+      // Disabling responding to typed messages at get location handler
```

Comment:
[Chain](https://stackoverflow.com/questions/13207927/switch-statement-multiple-cases-in-javascript?utm_medium=organic&utm_source=google_rich_qa&utm_campaign=google_rich_qa) these case statements since they are the same.

Reply:
What are these case statements doing exactly?

Reply:
Nice, this is new to me 👍

Reply:
Since we're presenting a drop down, we don't want input from the text field, that's why we're breaking the cases there. There's the various slackMessages.actions to handle input from the drop downs instead